doc/templating.rst

```diff
@@ -202,8 +202,24 @@ Overriding works like this::
 
 .. data:: css_files
 
-   Similar to :data:`script_files`, for CSS files.
+   Similar to :data:`script_files`, for CSS files::
 
+      {% set css_files = css_files + ["_static/mystyle.css"] %}
+
+.. data:: css_props
+
+   .. versionadded:: 1.4
+
+   An optional dict where you can specify ``alternate`` and/or ``title``
+   attributes for the css files. The keys are the css filenames, the values
+   are dicts themselves:
+
+      {% set _dummy = css_props.update( {"_static/mystyle.css": {"alternate":False, "title":"Default"} }) %}
```

Comment:
same as above.

However, `css_props` data structure information is useful for everyone.
Can you update please?